Mobile.BuildTools/Tasks/ScssProcessorTask.cs

```diff
@@ -57,20 +49,16 @@ public override bool Execute()
             return true;
         }
 
-        internal IEnumerable<string> GetFilesToProcess()
-        {
-            return from f in NoneIncluded
-                   where (Path.GetExtension(f) == ".scss" ||
-                          Path.GetExtension(f) == ".sass") &&
-                         !Path.GetFileNameWithoutExtension(f)
-                    .StartsWith("_", StringComparison.InvariantCulture)
-                   select f;
-        }
-
         private IEnumerable<TaskItem> ProcessFiles(IEnumerable<string> inputFiles)
         {
             foreach (var file in inputFiles)
             {
+                if (Path.GetFileName(file)[0] == '_')
```

Comment:
isn't this case already filtered out in line 39?

Reply:
technically I should be able to remove of the filters now that we're passing the correct collection from the start.